posts/2023/06/29/how-to-use-the-new-constructed-inventory-feature-in-aap-2.4.md

```diff
@@ -0,0 +1,178 @@
+---
+title: How to use the new Constructed Inventory Feature in Ansible Automation Platform 2.4
+author: Alan Rominger
+date: 2023-06-29 13:00:00
+slug: how-to-use-the-new-constructed-inventory-feature-in-aap-2.4
+category: awx
+tags: awx
+type: text
+---
+
+# The New Constructed Inventory Feature
+
+In this blog we introduced the idea for a new smarter way of handling inventory based on the Ansible constructed plugin. Now in Ansible Automation Platform 2.4, we have introduced this as a fully supported feature and this blog aims to introduce you to it! 
+
+Constructed inventory is the successor to the existing Smart Inventory feature, and  is now presented as another choice when creating an Inventory in Ansible Automation Platform controller. This will take a list of ‘normal’ inventories as input, perform user-defined operations, filter, and produce a resultant inventory with content from the input inventories.
```

Comment:
```suggestion
[Constructed inventory]( https://docs.ansible.com/automation-controller/4.4/html/userguide/inventories.html#ug-inventories-constructed) is the successor to the existing _Smart Inventory_ feature, and  is now presented as another choice when creating an **Inventory** in Ansible Automation Platform controller. This will take a list of ‘normal’ inventories as input, perform user-defined operations, filter, and produce a resultant inventory with content from the input inventories.
```